pkgs/development/compilers/dotnet/vmr.nix

```diff
@@ -38,8 +38,7 @@
 let
   llvmPackages = llvmPackages_20;
 
-  stdenv =
-    if llvmPackages.stdenv.hostPlatform.isDarwin then swiftPackages.stdenv else llvmPackages.stdenv;
+  stdenv = llvmPackages.stdenv;
```

Comment:
`swiftPackages.stdenv` used to be necessary to compile the swift bits, but it's not needed any more.

This broke because the default changed to llvm 21, which has some problems (https://github.com/dotnet/runtime/issues/119706) building dotnet.